pkg/deploy/kubernetes_image_puller.go

```diff
@@ -35,6 +34,11 @@ import (
 
 var imagePullerFinalizerName = "kubernetesimagepullers.finalizers.che.eclipse.org"
 
+type ImageKeyValue struct {
```

Comment:
would be nice to add a comment to the struct

Reply:
Sounds great. I added a comment and also changed the struct name for more clarity:
https://github.com/eclipse-che/che-operator/blob/3db42c1f1e5149a9bfbaf26e88d065c5220caf49/pkg/deploy/kubernetes_image_puller.go#L37-L41